.github/workflows/smoke-test.yml

```diff
@@ -11,13 +11,31 @@ jobs:
     name: smoke-tests-${{ github.event.inputs.environment }}
     runs-on: ubuntu-latest
     steps:
+
+      - uses: actions/checkout@v2
+
+      - name: Set up Ruby 2.7.2
+        uses: actions/setup-ruby@v1
+        with:
+          ruby-version: 2.7.2
```

Comment:
Would be good to get this from the .tool-versions or Gemfile. Let's think about that when we upgrade Ruby next though.